### DISCUSSION TITLE
Winget! Sources, Winstall, and updates

### DISCUSSION THREAD

**Discussion by iamjonny** [microsoft/winget-cli#4122] [General]
I've been using winget since I discovered it exists, but still using it manually, not done much other than a batch/ps script on launch of a sandbox to get stuff to be able to winget in windows sandbox...

I've just heard of winstall, forget exactly what it does, but whilst looking at powertoys I noticed the latest release (14 hours ago) is not yet available on winget.

What workflows are used by people to update the winget package info?
By that I mean do contributors usually receive notifications that watched repos have new releases (for downloading and managing github applications and packages via winget)

Or is it more automated? Maybe some CI/CD is used to notify or build (the config resources).

I'm wondering, if not already available, what's involved (requirements limitations etc) to make (more/some/all?!) winget packages be up to date.

`winget search powertoys` #0.77  - latest: [0.78](https://github.com/microsoft/PowerToys/releases/tag/v0.78.0)

Thanks!

**Comment by denelon** [comment]
Here is how [PowerToys](https://github.com/microsoft/PowerToys/blob/main/.github/workflows/package-submissions.yml) automated submissions.
